src/compiler/python_generator.cc

```diff
@@ -66,6 +66,9 @@ using std::vector;
 
 namespace grpc_python_generator {
 
+GeneratorConfiguration::GeneratorConfiguration()
```

Comment:
Public data members not good enough for ya, eh? If it's strongly desired, sure, that can happen.